doc/code_style/mom.h

```diff
@@ -84,7 +84,7 @@ class YourMom : public MassiveEntity {
 
 protected:
 	/** Let your mom buy something */
-	void buy(const Thing& item);
```

Comment:
We should decide for either **one form** regarding also the setting ` PointerAlignment`. I like both forms though as they give somehow to different information (e.g. left=space in mem), so would prefer the left alignment there.